src/components/UI/Card.vue

```diff
@@ -15,17 +15,11 @@ export default defineComponent({
 </script>
 
 <style lang="scss" module>
-.container {
-  width: 100%;
-  max-width: fit-content;
-  margin: 1em 0;
+.header {
   border: solid 1.5px #d9d9d9;
   border-collapse: collapse;
-  box-shadow: 0 2px 3px #dfe0d7;
-}
-.header {
   text-align: left;
   font-size: 15pt;
```

Comment:
```suggestion
  font-size: 1.25rem;
```
見落としてた単位はpxかremで統一してほしい